DXRMapFixups/DeusEx/Classes/DXRFixupParis.uc

```diff
@@ -119,6 +123,13 @@ function PreFirstEntryMapFixes()
         Spawn(class'PlaceholderItem',,, vectm(-1464,-1649.6,-197)); //Bathroom stall 1
         Spawn(class'PlaceholderItem',,, vectm(-1096.7,-847,-197)); //Bathroom stall 2
         Spawn(class'PlaceholderItem',,, vectm(-2093.7,-293,-161)); //Club back room
+
+        p.x = -1821.85;
+        p.y = -351.37;
+        p.z = -207.11;
+        p.w = 200.0;
+        SetAllLampsState(true, true, false, p); // the two Lamp3s on the desks near the back exit, but not the one where the accountant is
```

Comment:
```suggestion
        SetAllLampsState(true, true, false, vectm(-1821.85,-351.37,-207.11), 200); // the two Lamp3s on the desks near the back exit, but not the one where the accountant is
```

Reply:
Could be `SetAllLampsState(true, true, false, NewSphere(-1821.85, -351.37, -207.11, 200.0));`

Reply:
I'm well aware that is possible, but it's still not what I suggested